pkgs/applications/networking/instant-messengers/signal-desktop/default.nix

```diff
@@ -1,12 +1,23 @@
-{ callPackage }: builtins.mapAttrs (pname: attrs: callPackage ./generic.nix (attrs // { inherit pname; })) {
-  signal-desktop = {
+{ callPackage, stdenv }: builtins.mapAttrs (pname: attrs: callPackage ./generic.nix (attrs // { inherit pname; })) {
+  signal-desktop-aarch64 = rec {
```

Comment:
@Mic92 is that what you had in mind?

Reply:
Almost we can keep the same attribute name `signal-desktop` but just have two separate version variables, one for aarch64 and one for the rest.

Reply:
So just one `version-aarch64` and one `version`? And then depending on the target architecture I choose which version to use? So basically what I have done before, but just two separate version variables?

Reply:
Yes.

Reply:
Is it now as you have imagined it @Mic92? If not, could you please give me an example.